Makefile

```diff
@@ -40,6 +42,12 @@ dev-image: build
 image:
 	docker build -t $(IMAGE) .
 
+push-container-to-gcloud: image
+	gcloud auth configure-docker
+	docker push $(IMAGE)
```

Comment:
Here you can tag to IMAGE-GCLOUD and then push to IMAGE-GCLOUD.

Reply:
Fixed